.github/workflows/update_policies.yml

```
@@ -0,0 +1,71 @@
+on:
+  workflow_call:
+env:
+  GO_VERSION: '1.22'
+  GCS_BUCKET_PATH: gs://cerbos-policy-reearth-oss/policies
+
+jobs:
+  update-policies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)